### PR TITLE
🧪 [testing improvement] Add tests for getActivityStyle

### PR DESCRIPTION
**Pull request by kurousa**
This PR addresses the missing tests for the `getActivityStyle` function in `formatters/DefaultFormatter.js`.

### 🎯 What:
The testing gap for `getActivityStyle`, a pure function defining activity-specific emojis and colors, was addressed.

### 📊 Coverage:
The following scenarios are now tested:
- Standard activity types: `Walk`, `Run`, `VirtualRun`, `Ride`, `VirtualRide`, `Swim`, `Hike`, `Workout`, `WeightTraining`.
- Edge case: Unknown activity types (returning the default style).

### ✨ Result:
Increased test coverage for activity styling logic and fixed a typo in the `DefaultFormatter` test file name.

---
*PR created automatically by Jules for task [16968422826251995655](https://jules.google.com/task/16968422826251995655) started by @kurousa*

### CODE REVIEW

File: tests/DefaultFormatter.spec.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { describe, it, expect } from 'vitest';
+import { makeDefaultDescription, getActivityStyle } from '../formatters/DefaultFormatter';
```
[the remark this answers is not in the document]
Comment (kurousa) [reply tests/DefaultFormatter.spec.js]: @jules check this
